src/layouts/Field/Field.js

```diff
@@ -125,15 +128,24 @@ class Field extends React.Component {
   static styles = styles;
 
   stylesFor = gridName => {
-    const { column, columnSpan } = this.props;
+    const { column, columnSpan, alignContentHorizontally } = this.props;
 
     const endColumn = column + columnSpan - 1;
+
+    const horizontalAlignment = {
+      top: 'start',
+      bottom: 'end',
+      center: 'auto',
+    };
+
     return {
       gridArea: `${gridName}${column} / ${gridName}${column} / ${gridName}${endColumn} / ${gridName}${endColumn}`,
       // align-self determines how the parts vertically align if an adjacent part is taller.
       // labels should stick to the bottom if an adjacent label has two or more lines.
       // content and helpText should stick to the top if an adjacent element is taller than them.
-      alignSelf: gridName === 'label' ? 'end' : 'start',
+      alignSelf: alignContentHorizontally
+        ? horizontalAlignment[alignContentHorizontally]
```

Comment:
Maybe it would make sense to use "auto" alignment for the field contents in general?

Reply:
we can do in that way, but then we won't know if the user has provided it, or it is a default value

Reply:
On second thought, it seems like we should default to the current behavior so we don't break anything. I know that `<textarea>` is a good example of something that shouldn't be centered with other fields.